etc/skydive.yml.default

```diff
@@ -253,5 +253,11 @@ ui:
   # extra_assets: /usr/share/skydive/assets
 
 k8s:
-  # The KUBECONFIG yml file, if k8sctl is executed inside the cluster, you may use the empty config file - kube-config-empty.yml
+  # The kube config file, if executed inside the cluster, you may use the empty
+  # file, kube-config-empty.yml
   # config_file: /etc/skydive/kube-cfg.yml
+  subprobes:
```

Comment:
Maybe we should put something in order to say that is experimental.

Reply:
@safchain - ok have added a comment in the above configuration file